src/arch/xtensa/Kconfig

```diff
@@ -17,6 +17,16 @@ config SMP
 	help
 	  Indicates that architecture uses multiple cores
 
+config HAVE_MASTER_CORE_ROM
+	bool "Have master core rom"
+	default y
```

Comment:
No help for this option ?

Reply:
No, it is not used anywhere, I was just asked to add it along with one for slave cores

Reply:
If it's not used then why add it, will it be used later ?

Reply:
Not sure, deleted